mala/datahandling/data_handler_base.py

```diff
@@ -148,7 +148,7 @@ def clear_data(self):
     # Loading data
     ######################
 
-    def _check_snapshots(self):
+    def _check_snapshots(self, comm=None):
```

Comment:
Using a parameter here on purpose and not `get_comm()`, as this makes it obvious at the call site that the function is collective.